packages/execution-environments/src/common/lockdown/lockdown-more.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable-next-line import/no-unassigned-import */
+import 'ses';
```

Comment:
Per my other comment, [`ses` should not be imported in this manner](https://github.com/MetaMask/snaps-skunkworks/pull/231/files#r807333562).